src/components/shared/MatchItem/MatchItem.styles.ts

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+
+import { PALETTE } from '@styles/palette';
+import { STYLES } from '@styles/styles';
+
+export const MatchItemWrapper = styled.div`
+  background-color: white;
+  display: flex;
+  padding: 12px;
+  gap: 10px;
+  border-radius: 8px;
+`;
+
+export const MatchStatus = styled.div`
+  ${STYLES.FLEX_CENTER}
+  flex-direction: column;
+  background-color: ${PALETTE.GRAY_100};
+  padding: 10px;
+  width: 82px;
+  height: 82px;
+  border-radius: 8px;
+`;
```

Comment:
theme사용해서 변경할 수 있습니다. 
```suggestion
export const MatchStatus = styled.div`
  ${({theme}) => theme.STYLES.FLEX_CENTER}
  flex-direction: column;
  background-color: ${({theme}) => theme.PALETTE.GRAY_100}
  padding: 10px;
  width: 82px;
  height: 82px;
  border-radius: 8px;
`;
```